{{cookiecutter.project_slug}}/_/deployment/python/deployment/kubernetes/database/overlays/development/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```

Comment:
This should be `bases`, right?

Reply:
`bases` is deprecated: https://github.com/kubernetes-sigs/kustomize/commit/cc531af66540bb312ccb494b5dbfa28738a0fde2#diff-22d99431b9d4cadea0076f0db632cde5